packages/access-api/src/utils/context.js

```diff
@@ -42,12 +41,12 @@ export function getContext(request, env, ctx) {
     env: config.ENV,
   })
 
-  const keypair = Signer.parse(config.PRIVATE_KEY)
+  const signer = configureSigner(config)
```

Comment:
can you do this inside the `loadConfig` ?

Reply:
👍 https://github.com/web3-storage/w3protocol/pull/275/commits/b9533c92d0b8080f9f6d14c240a89f370d6ad7f7